src/gui/sdl_mapper.cpp

```diff
@@ -2802,6 +2805,9 @@ void MAPPER_CheckEvent(SDL_Event *event)
 
 void MAPPER_HandleJoyDeviceEvent(SDL_JoyDeviceEvent* event)
 {
+	if (!SDL_TICKS_PASSED(event->timestamp, joystick_subsystem_init_timestamp)) {
```

Comment:
That's cool, just please put a comment above this that this is hack to get rid of the double-init log messages.

Reply:
done